libr/bin/format/mach0/mach0.c

```diff
@@ -3263,10 +3266,45 @@ static int parse_import_ptr(struct MACH0_(obj_t) *bin, struct reloc_t *reloc, in
 	return false;
 }
 
-struct import_t *MACH0_(get_imports)(struct MACH0_(obj_t) *bin) {
+static RBinImport *import_from_name(RBin *rbin, const char *orig_name) {
+	RBinImport *ptr = R_NEW0 (RBinImport);
+	if (!ptr) {
+		return NULL;
+	}
+
+	char *name = (char*) orig_name;
+	const char *_objc_class = "_OBJC_CLASS_$";
```

Comment:
this should be const char * const, otherwise the code below cannot be optimized because the string can be modified at any time. but imho it will look better to use defines or just inline the string in the startswith + strlen like its done for the `type` line